new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -48,8 +50,6 @@ const loadFile = async (fileSrc: FileSrcType)
 class LogFileManager {
     readonly #pageSize: number;
 
-    readonly #fileData: Uint8Array;
-
     readonly #fileName: string;
```

Comment:
fileName is set but not used. Shall we create a getter for it?

Reply:
It's used in #56 so I think it's fine to leave it in this PR.